src/styles/blocks/card.css

```diff
@@ -0,0 +1,80 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__name {
+  padding-top: 40px;
```

Comment:
Should be margin-bottom in prev element. Don't use padding for gaps between elements. For gap use margin-bottom and margin-right. Fix in all cases